### ISSUE TITLE
Most Prometheus metrics are erroneously displaying 0 values

### ISSUE THREAD

**Issue by Pete-LunaNova** [Fantom-foundation/go-opera#236]
**Describe the bug**
When Prometheus metrics are enabled most are erroneously displaying 0 values on the metrics exporter page.

**To Reproduce**
Run the fantom opera binary with the `--metrics` flag and navigate to the metrics endpoint hosted at the ip address of the node and the metrics port specified (default 19090)
Metrics should be listed in standard Prometheus format, however all the values for critical metrics such as `opera_chain:head:block` erroneously have the value `0`.

**Expected behavior**
These metrics should contain accurate values, such as those returned by the relevant RPC calls

**Additional context**
Prometheus metrics are the "gold standard" for effective node monitoring and alerting. Enabling validators to be able to rely on accurate prometheus metrics to monitor items like blockheight and its rate of progression should increase network robustness, as any issues can be detected and tackled promptly.

**Comment by rus-alex** [comment]
@Pete-LunaNova , which exactly go-opera version you tell about?
I didn't find default metrics port 19090 in the [last release](https://github.com/Fantom-foundation/go-opera/tree/release/1.0.2-rc.9).

Here is [the fix](https://github.com/Fantom-foundation/go-opera/pull/238) of last release to run node with metrics endpoint:
`> ./go-opera --metrics --metrics.addr=127.0.0.1 ...`
and get accurate (non zero) metric values. For example:
```
> curl http://localhost:6060/debug/metrics/prometheus | grep "head_block"
# TYPE chain_head_block gauge
chain_head_block 16
```

**Comment by Pete-LunaNova** [comment]
We're running 1.0.2-rc.5 on mainnet, which is the latest version on your releases page.

Sounds like you've been able to fix this, which is fantastic, thank you.

Do you have an ETA on when you are likely to push this release out for use on mainnet?

**Comment by quan8** [comment]
@Pete-LunaNova  fix is available in branch release/1.0.2-rc.9
